code/modules/mob/living/carbon/life.dm

```diff
@@ -425,3 +428,19 @@
 	adjustToxLoss(8)
 	if(prob(30))
 		to_chat(src, "<span class='notice'>You feel confused and nauseous...</span>")//actual symptoms of liver failure
+
+
+////////////////
+//BRAIN DAMAGE//
+////////////////
+
+/mob/living/carbon/proc/handle_brain_damage()
+	for(var/datum/brain_trauma/BT in traumas)
+		BT.on_life()
+
+	if(brainloss >= BRAIN_DAMAGE_DEATH) //rip

```

Comment:
People are going to whine about this a lot, and they will go out of their way to grief with this to declare that we should remove it. See past this. This is a good feature for the overhaul.

Reply:
I've already prevented most griefing by capping most sources of brain damage. Chems are the most surefire way of brain death, and that takes effort.